libs/fairness/src/lib/Controls/CalloutHelpBar.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IBounds } from "@responsible-ai/core-ui";
+import _ from "lodash";
+import { Stack, Toggle } from "office-ui-fabric-react";
+import React from "react";
+
+import { IErrorPickerPropsV2 } from "../FairnessWizard";
+import { SharedStyles } from "../Shared.styles";
+
+import { CalloutErrorBars } from "./CalloutErrorBars";
+import { CalloutGraph } from "./CalloutGraph";
+
+interface ICalloutHelpBarProps {
+  graphCalloutStrings: string[];
+  fairnessBounds?: Array<IBounds | undefined>;
```

Comment:
The ? already makes it possible to be undefined. Doing it twice can lead to weird situations and should be avoided.

Reply:
`Array<IBounds | undefined> | undefined` is not redundant because it allows for `IBounds[] | undefined[] | undefined`, where the key is the array of undefined items